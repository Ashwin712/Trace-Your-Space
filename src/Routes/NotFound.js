import React from "react";
import { Link } from "react-router-dom";
import PageNotFound from "../../shared/assets/PageNotFound.jpg";
const NotFound = () => (
  <>
    <img
      src={PageNotFound}
      alt="error_image"
      className="page_not_found"
    />
    <center>
      <Link to="/">Return to Home Page</Link>
    </center>
  </>
);
export default NotFound;
