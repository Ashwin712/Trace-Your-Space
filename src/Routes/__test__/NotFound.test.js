import React from "react";
import NotFound from "../NotFound";
import TestRenderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";

describe("Test for NotFound", ()=>{
    it('Snapshot should Match',()=>{
        const container = TestRenderer.create(<MemoryRouter><NotFound/></MemoryRouter>);
        expect(container.toJSON()).toMatchSnapshot();
    })
})
