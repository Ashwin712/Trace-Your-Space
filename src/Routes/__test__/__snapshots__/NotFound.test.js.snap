// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for NotFound Snapshot should Match 1`] = `
Array [
  <img
    alt="error_image"
    className="page_not_found"
    src="PageNotFound.jpg"
  />,
  <center>
    <a
      href="/"
      onClick={[Function]}
    >
      Return to Home Page
    </a>
  </center>,
]
`;
